### PR TITLE
Added ssl_password config option to KafkaProducer class, identical to…

### PR DESCRIPTION
**Pull request by kierkegaard13**
… option in KafkaConsumer class

### CODE REVIEW

**Comment by kierkegaard13** [comment]
This change adds an ssl_password option which allows you to create a KafkaProducer with SSL without needing to enter the password manually. This is identical to the ssl_password config option in the KafkaConsumer class.

File: kafka/client_async.py [hunk 1]
```diff
@@ -136,6 +136,8 @@ def __init__(self, **configs):
                 establish the certificate's authenticity. default: none.
             ssl_keyfile (str): optional filename containing the client private key.
                 default: none.
+            ssl_password (str) - optional password to be used when loading the
```
Comment (dpkp) [inline kafka/client_async.py]: nit: `:` instead of `-`

File: kafka/producer/kafka.py [hunk 2]
```diff
@@ -216,6 +216,8 @@ class KafkaProducer(object):
             establish the certificate's authenticity. default: none.
         ssl_keyfile (str): optional filename containing the client private key.
             default: none.
+        ssl_password (str) - optional password to be used when loading the
```
Comment (dpkp) [inline kafka/producer/kafka.py]: same re colon / dash

Comment (kierkegaard13) [inline kafka/producer/kafka.py]: ah my bad

Comment (kierkegaard13) [inline kafka/producer/kafka.py]: fixed

**Comment by kierkegaard13** [comment]
@dpkp can you retest this? All tests passed except for one, which was an issue with the kafka server I believe.

**Pull request merged**